### DISCUSSION TITLE
Description field for weekly working time

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3220] [General]
Is it possible to include the field description of the time recording in the "weekly working time" or to make the page configurable which fields of the time recording are displayed there ?

**Comment by kevinpapst** [comment]
No and I don't have plans to do so.

**Reply by niklasfink** [reply]
Hi @kevinpapst I would also find this very appealing to have as this is the easiest and fastest view to use and for most cases, me and my team just need to put duration and description for a time record.

I have tried to build it with the Custom Content plugin, but it fails due to the backend seemingly not accepting the description field in that view without something in addition.

Do you see a way how it could be possible with the Custom Content plugin or does it need some additional backend work? In the latter case, would you be open to a PR?

<details><summary>Example Screenshots:</summary>
<p>
<img width="1963" height="339" alt="image" src="https://github.com/user-attachments/assets/40f781ca-4996-47e3-9716-9d06980a8ebd" />
<img width="563" height="231" alt="image" src="https://github.com/user-attachments/assets/76554a6d-23bc-40e5-ba84-a7a31e23e0b5" />
</p>
</details> 

----

<details><summary>Btw, found a small UI bug:</summary>
<p>
<img width="879" height="455" alt="image" src="https://github.com/user-attachments/assets/1f80f44b-425e-42c5-9486-53cb907ab5ec" />
<img width="880" height="430" alt="image" src="https://github.com/user-attachments/assets/7c1f9773-e8b9-4caf-b759-7d2fddd7feda" />

I think it's missing the CSS styles from .input-group with the max-width styles.
section.quick-entry-page #quick_entry_box .duration-widget .input-group
</p>
</details>

**Reply by kevinpapst** [reply]
Thanks for sharing the bug, will be fixed with the next release. 
But its pure luck that I saw it: please, please, please - do not mix topics in single posts.

**Reply by kevinpapst** [reply]
Whether I accept a PR highly depends on the quality of it. 
I have no real concept how it might become reality, but I doubt that adding input fields below is a good solution. 
Likely needs some kind of interactive element that can be shown and hidden. 
Someone suggested a tiny clickable corner element in the duration input, which opens a modal to enter the description. 
Or some kind of toggle to display and hide the fields. 

Lets discuss possible solutions first, before you start building.

**Reply by marcelbick** [reply]
Hi @kevinpapst ,
I also think a feature like this would be very beneficial to have, as it gives the end-user a quick way to add descriptions to tasks. Additionally, I think that having the option to add tags would be very helpful as well. 
However, instead of adding the description / tags on a daily basis, I would suggest it as their own respective columns, meaning you enter a description for a given line / activity for all days. 
I played around a bit with the Custom Content plugin in the demo and came up with the following:

<img width="945" height="324" alt="grafik" src="https://github.com/user-attachments/assets/d15ba108-1feb-4478-a166-d1d23b3dbdfc" />

Do you have any plans to add something like this or develop a plugin that features this? My company would be very keen to have a feature like this.
